examples/tui_keys.rs

```diff
@@ -1,5 +1,7 @@
 use dioxus::events::WheelEvent;
 use dioxus::prelude::*;
+use dioxus_html::geometry::ScreenPoint;
+use dioxus_html::input_data::MouseButtonSet;
```

Comment:
We will probably re-export these through the html namespace or globally. Not a requirement for this PR, but something I will be doing when cleaning up for 0.3